response_operations_ui/controllers/case_controller.py

```diff
@@ -173,39 +143,3 @@ def generate_iac(case_id):
         raise ApiError(response)
 
     return response.json()['iac']
-
-
-def get_iac_count_for_case(case_id):
-    url = get_iac_url(case_id)
-
-    response = requests.get(url=url, auth=app.config['CASE_AUTH'])
-
-    try:
-        response.raise_for_status()
-    except requests.HTTPError:
-        logger.exception('Error getting IAC count', case_id=case_id, url=url)
-        raise ApiError(response)
-
-    iac_count = len(response.json())
-
-    logger.debug("IAC count for case", case_id=case_id, url=url, iac_count=iac_count)
-
-    return iac_count
-
-
-def get_case_events_by_case_id(case_id):
```

Comment:
This is actually needed as part of my code for the view response time

Reply:
Whereabouts is that code mate?